doc/api/globals.md

```diff
@@ -156,6 +156,24 @@ added: v10.0.0
 
 The WHATWG `URLSearchParams` class. See the [`URLSearchParams`][] section.
 
+## TextEncoder
+<!-- YAML
+added: v11.0.0
+-->
+
+<!-- type=global -->
+
+The WHATWG `TextEncoder` class. See the [`TextEncoder`][] section.
```

Comment:
If what we're doing is migrating from `util` to a global, I think it'd be better to have the reverse: document here and have `util.TextEncoder` link to the definition here.